config/webpack.config.js

```diff
@@ -351,14 +361,16 @@ module.exports = function (webpackEnv) {
             {
               test: /\.(js|mjs|jsx|ts|tsx)$/,
               include: paths.appSrc,
-              loader: require.resolve("babel-loader"),
+              loader: "babel-loader",
```

Comment:
Did this work?

The `test: /parquet\.cjs\.js/,` used the require.resolve version. So perhaps either way works?

Reply:
Yes either way works.